syntax = "proto3";

package evaluation;
 
service Environment{
  rpc get_action_space(Package) returns (Package) {}
  rpc act_on_environment(Package) returns (Package) {}
}
 
message Package{
  bytes SerializedEntity = 1;
}
